packages/ui/src/kit/VInput/VInput.vue

```diff
@@ -91,6 +121,16 @@ export default {
     isError() {
       return !!this.error;
     },
+
+    isResetAllowed() {
+      return this.isResetable && !this.disabled;
+    },
+  },
+
+  methods: {
+    onResetClick() {
+      this.$emit('reset');
```

Comment:
for what emit reset? maybe change value of input to empty string?

Reply:
> maybe change value of input to empty string?

we always emit event upwards, so nope
and `value` is prop, it's readonly, `sync` is bad idea as well